Client/Frontend/Browser/BrowserViewController/BVC+Rewards.swift

```diff
@@ -168,9 +168,13 @@ extension BrowserViewController: RewardsUIDelegate {
     func loadNewTabWithURL(_ url: URL) {
         self.presentedViewController?.dismiss(animated: true)
         
-        let request = URLRequest(url: url)
-        let isPrivate = PrivateBrowsingManager.shared.isPrivateBrowsing
-        tabManager.addTabAndSelect(request, isPrivate: isPrivate)
+        if let tab = tabManager.getTabForURL(url) {
+            tabManager.selectTab(tab)
```

Comment:
Will this work when you are in private mode, and the found tab is in normal mode? what happens then

Reply:
You can't access the Brave Rewards panel from Private Mode, therefore I don't think this edge case is possible.